src/scripts/main.js

```diff
@@ -1,3 +1,262 @@
 'use strict';
 
 // write your code here
+const body = document.querySelector('body');
+
+const rows = body.querySelectorAll('tr');
+const button = document.querySelector('button');
+const tableScore = body.querySelector('.game-score');
+const messages = body.querySelectorAll('.message');
+const winMessage = body.querySelector('.message-win');
+const loseMessage = body.querySelector('.message-lose');
+const cells = body.querySelectorAll('.field-cell');
+
+const board = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+];
+const newNumber = [2, 2, 2, 2, 2, 2, 2, 2, 4, 2];
```

Comment:
Consider using a function that returns a random number based on a probability (90% for 2 and 10% for 4) instead of the 'newNumber' array.